binder/binder.pro

```diff
@@ -0,0 +1,19 @@
+TEMPLATE        = lib
+CONFIG         += plugin
+INCLUDEPATH    += ../
+HEADERS         = binder.h \
+                  gnss-binder-types.h
+SOURCES         = binder.cpp
+TARGET          = $$qtLibraryTarget(geoclue-hybris-binder)
+DESTDIR         = ../
+
+QT = core network
+
+CONFIG += link_pkgconfig
+PKGCONFIG += android-headers connman-qt5 systemsettings libgbinder libglibutil gobject-2.0 glib-2.0
+
+# install
+target.path = /usr/libexec
+INSTALLS += target
+
+CONFIG += install_ok  # Do not cargo-cult this!
```

Comment:
Do not cargo-cult this :)  I don't think this `install_ok` thing is necessary.

Reply:
I used the Qt example as the base for the project file, I'll remove unneeded lines.